packages/jaeger-ui/src/components/TraceDiff/TraceDiffGraph/TraceDiffGraph.css

```diff
@@ -47,6 +47,37 @@ limitations under the License.
   stroke-width: 0.7;
 }
 
+/* Find within diff */
+.TraceDiffGraph--uiFind {
+  border: solid 1px #a9a9a9;
+  border-radius: 21px;
+  bottom: 20px;
+  cursor: auto;
+  display: flex;
+  padding: 5px;
+  position: absolute;
+  right: 20px;
+  z-index: 1;
+}
+
+.TraceDiffGraph--uiFind--input {
+  width: 300px;
+  transition: ease 1s;
+  margin: 0px 3px;
+}
+
+.TraceDiffGraph--uiFind:not(:hover) > .TraceDiffGraph--uiFind--input:not(:focus) {
+  border: none;
+  padding: 0px;
+  width: 0;
+  margin: 0px;
+}
+
+.TraceDiffGraph--uiFind--icon {
+  cursor: pointer;
+  font-size: 32px;
+}
+
```

Comment:
What do you think of the styling, below, for the find input? 

<img width="484" alt="trace-diff-find-input" src="https://user-images.githubusercontent.com/2304337/52762208-9fb50380-2fe4-11e9-86a6-dc1ea5a9a167.png">

I would say persistent or maybe reduced width but not totally collapsed might work. One nice thing about it being persistent is the search term would always be visible.

Reply:
The reason I had it hide was in case it covered a node. I've since realized that the user can pan "past" the graph so that nodes at the border of the graph area can be centered (or at least uncovered). Once I realized that I already had the resizing so I kept it since it seemed nice and out of the way. But expanded width when the filter text is not-empty is certainly helpful. I'll change from a large Icon to the Find text, add the count, then only show reduced (not zero) width when the text is empty.

Reply:
I dropped the `Find` part in response to @copa2 's feedback to have a more consistent look between the `DiffGraph` uiFind and the `TracePageSearchBar`. 

Empty:
![image](https://user-images.githubusercontent.com/3471184/53832895-5574d180-3f55-11e9-9e8a-5afdda3ecf54.png)
With text:
![image](https://user-images.githubusercontent.com/3471184/53833253-13985b00-3f56-11e9-8996-fa9f938b5f05.png)